examples/windows/windows-perfmon-proc-owner.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+Powershell -File "C:\Program Files\New Relic\newrelic-infra\integrations.d\windows-perfmon-proc-owner.ps1"
```

Comment:
please remove the batch file (`.bat`) dependency here and add the `shell` attribute to the config so you can execute the PowerShell script natively. Example [HERE](https://github.com/newrelic/nri-flex/blob/master/examples/windows/windows-ping-example.yml)

Reply:
done